torchvision/transforms/v2/_color.py

```diff
@@ -177,7 +177,22 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return output
 
 
-# TODO: This class seems to be untested
+class RandomPermuteChannels(Transform):
+    _transformed_types = (
+        datapoints.Image,
+        PIL.Image.Image,
+        is_simple_tensor,
+        datapoints.Video,
+    )
+
+    def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
+        num_channels, *_ = query_chw(flat_inputs)
+        return dict(permutation=torch.randperm(num_channels).tolist())
```

Comment:
why does this need to be a list? Wasn't it was kept as a tensor in the previous implem?

Reply:
I've checked, and we can indeed leave it as tensor even with JIT. The reason I have used a list here is that we have very few functionals that take a tensor as parameter. Since we are never going to script the transform here, we could also have the annotation on the dispatcher as `List[int]`, but pass a tensor from the transform. No strong opinion. Up to you.

Reply:
I don't really mind what the annotation is, I just want to make sure we still support tensors so that users don't have to call tolist all the time